spec/features/idv/in_person_spec.rb

```diff
@@ -654,4 +654,25 @@ def it_captures_address_with_state_id
       end
     end
   end
+
+  context 'in_person_capture_secondary_id_enabled feature flag enabled and same address as id',
+          allow_browser_log: true do
+    let(:user) { user_with_2fa }
+
+    it 'skips the address page' do
+      allow(IdentityConfig.store).to receive(:in_person_capture_secondary_id_enabled).
```

Comment:
Nit: I would recommend putting this `IdentityConfig.store` setup in a `before` block.

Reply:
Done.